errors/index.js

```diff
@@ -5,8 +5,9 @@ exports.handleCustomErrors = (err, req, res, next) => {
 };
 
 exports.handlePsqlErrors = (err, req, res, next) => {
+    const invalid_text_representation = '22P02';
     const foreign_key_violation = '23503';
-    if (err.code === foreign_key_violation) {
+    if (err.code === foreign_key_violation || err.code === invalid_text_representation) {
```

Comment:
I really like this in your error handler, we can hope people know what the codes mean but these are really good variable names.